osu.Game/Screens/Edit/Verify/IssueTable.cs

```diff
@@ -88,10 +86,10 @@ private TableColumn[] createHeaders()
             var columns = new List<TableColumn>
             {
                 new TableColumn(string.Empty, Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize)),
-                new TableColumn("Type", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize, minSize: 60)),
-                new TableColumn("Time", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize, minSize: 60)),
-                new TableColumn("Message", Anchor.CentreLeft),
-                new TableColumn("Category", Anchor.CentreRight, new Dimension(GridSizeMode.AutoSize)),
+                new TableColumn(@"Type", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize, minSize: 60)),
+                new TableColumn(@"Time", Anchor.CentreLeft, new Dimension(GridSizeMode.AutoSize, minSize: 60)),
+                new TableColumn(@"Message", Anchor.CentreLeft),
+                new TableColumn(@"Category", Anchor.CentreRight, new Dimension(GridSizeMode.AutoSize)),
```

Comment:
Any reason for making these verbatim strings, along with a few others in this diff? This stops the analyser from finding these strings for future localisation.

Reply:
Curious which analyser is showing this. For us they are only hint-level.